types/react-big-calendar/index.d.ts

```diff
@@ -240,7 +246,7 @@ export class DateLocalizer {
     format(value: FormatInput, format: string, culture: Culture): string;
 }
 
-export interface BigCalendarProps<TEvent extends Event = Event, TResource extends object = object> extends React.Props<BigCalendar<TEvent, TResource>> {
+export interface BigCalendarProps<TEvent extends Event = Event, TResource extends object = object> {
```

Comment:
permalink: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6f7a141fb85d645316c66eb228f158a3f24513c9/types/react/index.d.ts#L1228

Reply:
I made this originally but I'm not sure why. What happens if you don't remove this?

Reply:
Extending `Props` is fine -- it's not an error to have it.  It was required in some earlier version of the react typings and previous typescript version (I don't know what), but is now marked as deprecated. I suppose you might want to keep to support older versions of the react typings and/or typescript.

Reply:
I'm mostly concerned about the `ref` part. If it isn't a requirement for this PR we should discuss it in a separate PR. It should include type tests and runtime tests to verify that the ref matches. Runtime test could just be a codesandbox for manual testing.

Reply:
OK, I don't have strong feelings about eliminating React.Props.  I'll put it back.

Reply:
Ok, the `extends React.Props<>` is put back.